docs/source/docker/integration/dagster_aws.rst

```diff
@@ -484,8 +378,8 @@ To put the solids together in a single pipeline, save the following code in `aws
         
         """
         file_name = download_random_video_from_pexels()
-        object_name = upload_video_to_s3(file_name)
-        run_lightly_onprem(object_name)
+        upload_video_to_s3(file_name)
+        run_lightly_onprem()
```

Comment:
I was wondering why there is no argument passed to running lightly on prem. Perhaps you could tell that lightly analyses all videos in the bucket, not only the one just uploaded?